python-lib/tagger.py

```diff
@@ -1,9 +1,110 @@
+"""Module to prepare the tagging of the documents
+splits on sentences and instanciates a Matcher to tag the documents"""
+
+from spacy_tokenizer import MultilingualTokenizer
+from dkulib_io_utils import generate_unique
+from spacy.matcher import PhraseMatcher
+from fastcore.utils import store_attr
+from typing import AnyStr
 import pandas as pd
 
 
 class Tagger:
-    def __init__(self, settings):
-        pass
+    def __init__(
+        self,
+        text_df,
+        ontology_df,
+        text_column,
+        language,
+        language_column,
+        tag_column,
+        category_column,
+        keyword_column,
+        lemmatize,
+        case_sensitive,
+        normalize,
+        mode,
+    ):
+        store_attr(but=["text_df", "ontology_df"])
+        self.text_df = text_df.get_dataframe()
```

Comment:
Two comments on class structure:
- I would rather have text_df as an argument to the tag method, instead of a class attribute. That way you can use the same class instance to tag multiple data frames, without re-instantiating. 
- Handling ontology as a data frame is clunky. We do have to read it as a data frame, as it's stored this way in DSS. But overall, it's not a good format. How about moving the `_get_patterns` method used to transform this data frame into DkuConfigLoadingOntologyTagging?